core/new-gui/src/app/workspace/service/workflow-graph/model/workflow-action.service.ts

```diff
@@ -536,6 +553,10 @@ export class WorkflowActionService {
     this.executeStoreAndPropagateCommand(command, commandMessage);
   }
 
+  public deleteCommentBox(commentBoxID: string): void {
+    this.deleteCommentBoxInternal(commentBoxID);
```

Comment:
Same here, these two actions need to send a `CommandMessage` to propagate actions to other clients. However, for now, the name of this function `deleteCommentBox` and `addCommentBox` should be added to `commandFuncs` in `command.interface.ts`

Another thing is maybe this action should also have undo/redo?